src/NuGet.Clients/NuGet.Console/GlobalSuppressions.cs

```diff
@@ -25,3 +25,7 @@
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetConsole.Implementation.Console.WpfConsole.Activate")]
 [assembly: SuppressMessage("Reliability", "CA2016:Forward the 'CancellationToken' parameter to methods", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetConsole.ChannelOutputConsole.WriteToOutputChannelAsync(System.String,System.String,System.String,System.Threading.CancellationToken)~System.Threading.Tasks.Task")]
 [assembly: SuppressMessage("Usage", "CA2213:Disposable fields should be disposed", Justification = "<Pending>", Scope = "member", Target = "~F:NuGetConsole.Implementation.PowerConsoleToolWindow._consoleParentPane")]
+[assembly: SuppressMessage("ApiDesign", "RS0030:Do not used banned APIs", Justification = "<Pending>", Scope = "member", Target = "~F:NuGetConsole.ChannelOutputConsole._pipeLock")]
+[assembly: SuppressMessage("ApiDesign", "RS0030:Do not used banned APIs", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetConsole.ChannelOutputConsole.WriteToOutputChannelAsync(System.String,System.String,System.String,System.Threading.CancellationToken)~System.Threading.Tasks.Task")]
```

Comment:
I am curious why are we using banned API.

Reply:
I'm not sure, though it looks like they have been suppressed inline.